[web]/ajax/ajax.js

```diff
@@ -137,18 +135,16 @@ function ElementManager() {
 	}
 }
 
-var resourceManager = new ResourceManager();
+class ResourceManager {
+	constructor() {
+		this.resources = new Array();
+	}
 
-function ResourceManager() {
-	this.resources = new Array();
-	this.get = function (name) {
-		for ( i = 0; i < this.resources.length; i++ )
-		{
+	get(name) {
+		for (let i = 0; i < this.resources.length; i++) {
 			var resource = this.resources[i];

```

Comment:
Prefer [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of).
```suggestion
		for (const resource of this.resources) {
```